lasagne/layers/conv.py

```diff
@@ -48,7 +49,7 @@ def __init__(self, incoming, num_filters, filter_length, stride=1,
             self.nonlinearity = nonlinearity
 
         self.num_filters = num_filters
-        self.filter_length = filter_length
+        self.filter_size = filter_size
```

Comment:
For consistency with the 2D (and possibly in the future 3D) case, I think it would be nice to do `as_tuple(filter_size, 1)` here, so that the `filter_size` argument can also accept single element tuples. This will also require some other modifications to the code, as it is currently assumed to be an integer (and it would become a single element tuple).